firmware/spade/src/rpi/main.c

```diff
@@ -239,6 +263,8 @@ int main() {
   }
 
   // Start a core to listen for keypresses.
+  multicore_reset_core1();
+  sleep_ms(100);
```

Comment:
why was this sleep added?

Reply:
Iirc this was part of a jank fix for keypresses not working while debugging in GDB - have you had this issue? (either way it shouldn't be here, though)

Reply:
Was a fix for multicore not working while debugging in GDB. Line 224 of this file is also a debug-mode related fix for timers not working while in GDB. Should I extract these to a separate PR @grymmy?

Reply:
almost certainly should be separate if it can be described as thus (which is different than sd card support)